generate_print_files.py

```diff
@@ -91,10 +91,19 @@ def generate_print_file(print_file_path: Path, uac_qid_links, config):
     with io.StringIO() as print_file_stream:
         csv_writer = csv.DictWriter(print_file_stream, fieldnames=PRINT_FILE_TEMPLATE, delimiter='|')
         row_count = 0
-        for row_count, result_row in enumerate(uac_qid_links, start=1):
-            print_row = {'UAC': result_row['uac'], 'QUESTIONNAIRE_ID': result_row['qid'],
-                         'PRODUCTPACK_CODE': config["Pack code"]}
-            csv_writer.writerow(print_row)
+
+        if config['Pack code'].startswith('D_CCS'):
```

Comment:
I think this function is looking a bit too busy/nested.
I've had a play with a few ideas to remove some of the nest by deciding on the formatting and setting a function for this?  Happy to defer to Pythonista opinion on this though